### PR TITLE
Update Markdown doc

### PR DESCRIPTION
**Pull request by TimKam**
Minor doc update after Eric released recommonmark 0.5.0, see: https://github.com/rtfd/recommonmark/pull/113 / https://pypi.org/project/recommonmark/ :-)

Can you merge, @tk0miya?

### CODE REVIEW

File: doc/usage/markdown.rst [hunk 1]
```diff
@@ -24,16 +24,12 @@ To configure your Sphinx project for Markdown support, proceed as follows:
 
 #. Install the Markdown parser *recommonmark* from its source on GitHub::
```
Comment (nsoranzo) [inline doc/usage/markdown.rst]:
```suggestion
#. Install the Markdown parser *recommonmark*::
```

Comment (jfbu) [reply doc/usage/markdown.rst]: @nsoranzo thanks for suggested edit, ok (I don't understand why the diff shows a removed space before the `#` character as there is none in original)

**Review by jfbu — CHANGES_REQUESTED** [review]
can you please incorporate @nsoranzo edit ? thanks :)

**Comment by TimKam** [comment]
Sorry, for forgetting this...

**Review by jfbu — APPROVED** [review]
LGTM!

**Pull request merged**